uuyeong/steam_crawler.py

```
@@ -0,0 +1,127 @@
+            'description': '',
+            'requirements': ''
+        }
+        
+        # 리뷰 정보 수집
+        try:
+            reviews_elem = driver.find_elements(By.CSS_SELECTOR, '.user_reviews_summary_row')
+            reviews_text = []
+            for review in reviews_elem[:3]:  # 상위 3개만
+                reviews_text.append(review.text)
+            game_info['reviews'] = ' '.join(reviews_text)
+        except:
```

Ruff (0.13.2)

Do not use bare `except`

(E722)